includes/utilities.inc

```diff
@@ -206,30 +211,32 @@ function islandora_entities_get_related_pids($identifier, $title, $type) {
       'islandora_entities_thesis_base_query',
       '+RELS_EXT_hasModel_uri_mt:"ir:thesisCModel"'),
   );
-  $query = "+mods_name_personal_displayForm_mt:\"($identifier)\"  {$base_query[$type]}";
+  $query = "+mods_name_personal_displayForm_mt:\"($identifier)\" AND {$base_query[$type]}";
 
   $params = array(
     'fl' => 'dc.title, mods_name_personal_displayForm_mt, PID',
-    'sort' => variable_get('islandora_entities_query_sort', ''),
   );
+  if (strlen(variable_get('islandora_entities_query_sort', '')) > 0) {
```

Comment:
Could we grab the variable once here before doing the check, and possibly use `!empty()` instead of doing a direct comparison on string length?